plugins/sudoers/toke.l

```diff
@@ -1133,14 +1133,14 @@ strlcpy_expand_host(char *dst, const char *src, size_t size)
 		n = size ? size - 1 : 0;
 	    }
 	    dst += n;
-	    size -= n;
```

Comment:
There is no need to change the order here.  I prefer it the way it currently is.